### PR TITLE
Changed submit from 'enter' to submit button

### PR DESCRIPTION
**Pull request by jackfrancismurphy**

### CODE REVIEW

File: newdle_frontend.js [hunk 1]
```diff
@@ -43,19 +44,15 @@ function onSubmit(){
...
-        }
-    }
-);
 
 function getHeadlines(headlines_json){
-    document.getElementById("presented_headline").innerText = headlines_json.scrambled_headline
-    headlineArray = headlines_json.headline.split(" ")
-    headlineArrayLower = headlines_json.headline.toLowerCase().split(" ")
+    console.log(headlines_json)
+    document.getElementById("presented_headline").innerText = "Transfer news LIVE: Barcelona 'preparing' Lewandowski dib with Arsenal, Man Utd and Liverpool latest - The Mirror"
```
Comment (JackDanielHarding) [inline newdle_frontend.js:53]: Have you left in a permentant headline for testing here?

Comment (JackDanielHarding) [reply newdle_frontend.js:53]: It also starts completely solved?

**Review by JackDanielHarding — CHANGES_REQUESTED** [review]

File: newdle_frontend.js (hunk 1 above)
Comment (jackfrancismurphy) [reply newdle_frontend.js:53]: That's weird, I put this back afterwards but seem to have pushed before I reverted the testing coding?

**Review by jackfrancismurphy — COMMENTED** [review]
Pushed without silly test code

**Review by JackDanielHarding — APPROVED** [review]

**Pull request merged**